plugins/kpexec.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kpexec
+spec:
+  homepage: https://github.com/ssup2/kpexec
+  shortDescription: Execute process with privileges in a pod
+  description: |
+    kpexec is a kubernetes cli that runs commands in a container with high
+    privileges. kubectl-exec runs the command with the same privileges as the
+    container. For example, if a container does not have network privileges, the
+    command executed by kubectl-exec also has no network privileges. Also,
+    kubectl-exec does not provide an option to force command to run with high
+    privileges. This makes debugging the pod difficult.
+
+    kpexec helps execute pod debugging smoothly by executing commands with high
+    privileges regardless of container privileges. Also, kpexec supports tools
```

Comment:
Maybe add something like this:

`kpexec` runs a highly privileged container on the same node as the target container, and joins into the namespaces of the target container (IPC, UTS, PID, net, user, mount).